pkg/runtime/upgrade.go

```diff
@@ -199,3 +207,28 @@ func (k *KubeadmRuntime) tryUncordonNode(ip, nodename string) error {
 	}
 	return nil
 }
+
+func (k *KubeadmRuntime) changeCRIVersion() error {
+	if err := modifyFile("/etc/image-cri-shim.yaml", "version: v1alpha2", "version: v1"); err != nil {
```

Comment:
1. Does this just seem to be a local modification? What if the node executed by sealos is not in the cluster?
2. Image-cri-shim is executed at each node, and the modification should modify all nodes.